ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/FSharpErrorUtil.fs

```diff
@@ -116,6 +116,10 @@ let getLambdaCanBeReplacedWarningText (replaceCandidate: IFSharpExpression) =
         sprintf "Lambda can be replaced with '%s'" x.QualifiedName
     | _ -> "Lambda can be simplified"
 
+let getIfCanBeReplacedByConditionOperandWarningText needNegation =
+    if needNegation then "'if' expression can be replaced by condition operand negation"
+    else "'if' expression can be replaced by condition operand"
```

Comment:
`with` seems to suit better than `by` to me, although I'm not a native speaker.
https://english.stackexchange.com/questions/151309/replace-with-versus-replace-by

Reply:
I did it by analogy with C#

![image](https://user-images.githubusercontent.com/26364714/119652866-4f69b480-be2f-11eb-8e61-af9dd2661f3f.png)

Reply:
I think this one would be more similar to our case:
<img width="529" alt="Screenshot 2021-05-26 at 14 37 43" src="https://user-images.githubusercontent.com/3923587/119653527-1b42c380-be30-11eb-969c-c774cec13a82.png">

Though, it's sad it's not very consistent in C# support.